packages/entities/entities-routes/sandbox/pages/RouteFormPage.vue

```diff
@@ -91,3 +117,29 @@ const onUpdate = (payload: Record<string, any>) => {
   router.push({ name: 'route-list' })
 }
 </script>
+
+<style lang="scss" scoped>
+.sandbox-route-flavor-control {
+  :deep(.k-collapse) {
+    &.is-collapsed {
+      .k-collapse-heading {
+        margin-bottom: 0 !important;
```

Comment:
Can we please use tokens everywhere in the styles ?

Reply:
Thanks. I've updated the PR.